### PR TITLE
Footer editable fields & separation

### PR DESCRIPTION
**Pull request by slee288**
### WHAT IT DOES:

- Add editable fields on Atelier admin for footer
- Separate campaign footer from the global footer

### WHAT TO TEST:

- [ ] On Atelier **Settings** tab, go to the section **Store Footer** to edit the footer links or footer background color.
- [ ] Go to any campaign page, and check if the settings have been applied on the footer.
- [ ] Go to the root website (no subdomain), and check if the global footer remains the same

### PREVIEW:

- Store Footer fields

image

- Global footer

image

- Campaign footer

image

### CODE REVIEW

File: components/atelier-landing-page/Footer/index.tsx [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+              Terms and Conditions
+            </Link>}
+            {themeSettings.footer.footerFAQLink && <Link href={themeSettings.footer.footerFAQLink}>
+              FAQ
+            </Link>}
+          </div>
+        </div>
+      </div>
+    </footer>
+  ) : (
```
Comment (pratikmdhr) [inline components/atelier-landing-page/Footer/index.tsx:34]: we should probably keep the store related code separate from our atelier.sale landing page. Currently the components inside `atelier-landing-page` only includes the code for our landing page which is controlled by Tina CMS.

We could probably create a new footer component inside `campaign`?

**Review by pratikmdhr — CHANGES_REQUESTED** [review]

File: components/atelier-landing-page/Footer/index.tsx (hunk 1 above)
Comment (slee288) [reply components/atelier-landing-page/Footer/index.tsx:34]: now I think about it I think this solution might be better, so should I separate it inside `campaign` directory?

Comment (pratikmdhr) [reply components/atelier-landing-page/Footer/index.tsx:34]: hmm are also planning to add a separate landing page for each store?

Comment (pratikmdhr) [reply components/atelier-landing-page/Footer/index.tsx:34]: actually lets huddle lol

Comment (slee288) [reply components/atelier-landing-page/Footer/index.tsx]: @pratikmdhr I separated the footer on different directories - let me know if you have a chance to review it

**Review by pratikmdhr — APPROVED** [review]
looks good ! ✅

**Pull request merged**